### ISSUE TITLE
Address growing backlog of Report-Site-Issue addon issues

### ISSUE THREAD

**Issue by wisniewskit** [mozilla/webcompat-team-okrs#234]
Basically aim to resolve bugs such as:
- https://bugzilla.mozilla.org/show_bug.cgi?id=1697124
- https://bugzilla.mozilla.org/show_bug.cgi?id=1677415
- https://bugzilla.mozilla.org/show_bug.cgi?id=1560449
- https://bugzilla.mozilla.org/show_bug.cgi?id=1400263
- https://bugzilla.mozilla.org/show_bug.cgi?id=1318353
- rough time estimate: 1 weeks of engineering effort, not counting any surprises or unforeseen delays.

**Comment by denschub** [comment]
So, about [bug 1677415](https://bugzilla.mozilla.org/show_bug.cgi?id=1677415):

This is a timing issue that, if we want to resolve it properly, needs quite a bit of rework on both the reporter, but also on the webcompat.com end to make it accept the pre-fill data on a different channel. All that doesn't look like a quick fix to me, and the 1 week estimate is, in my opinion, a bit too close if this issue is included. WDYT? :)

**Comment by wisniewskit** [comment]
@denschub: Yes, in hindsight you're right, I didn't really count the webcompat.com work in my estimate, which was a bit silly. Do you suppose doubling the estimate is reasonable?

**Comment by denschub** [comment]
@ksy36 is a way better person to estimate the webcompat.com work, but I think two weeks are enough, yeah.

**Comment by karlcow** [comment]
@wisniewskit is it 1 week per issue or 1 week for the total

**Comment by wisniewskit** [comment]
@karlcow , at this stage I feel it will be about two weeks total, though @ksy36 might have additional insight.
